src/topology/locally_constant/basic.lean

```diff
@@ -222,8 +223,24 @@ coe_injective (funext h)
 theorem ext_iff {f g : locally_constant X Y} : f = g ↔ ∀ x, f x = g x :=
 ⟨λ h x, h ▸ rfl, λ h, ext h⟩
 
-protected lemma continuous [topological_space Y] (f : locally_constant X Y) : continuous f :=
-f.is_locally_constant.continuous
+section codomain_topological_space
+
+variables [topological_space Y] (f : locally_constant X Y)
+
+protected lemma continuous : continuous f := f.is_locally_constant.continuous
+
+/-- We can turn a locally-constant function into a bundled `continuous_map`. -/
+def to_continuous_map : C(X, Y) := ⟨f, f.continuous⟩
+
+instance : has_coe (locally_constant X Y) C(X, Y) := ⟨to_continuous_map⟩
```

Comment:
This instance should have a lemma that establishes a simp normal form.

Reply:
Isn't that the lemma below this? What do you mean?

Reply:
We need a lemma either that states `(f : C(X, Y)) = f.to_continuous_map` or vice versa, otherwise we have two ways to spell the same thing neither of which is obviously canonical.

Reply:
```suggestion
/-- As a shorthand, `locally_constant.to_continuous_map` is available as a coercion -/
instance : has_coe (locally_constant X Y) C(X, Y) := ⟨to_continuous_map⟩
```